cargo psp issue

Is rust-psp not supported on Windows or did I do something wrong? 

> I am using the rustup-init.exe(64-bit) installer

toolchain info:

```
PS C:\Users\diamant3\Desktop\psp-controller> rustup show  
Default host: x86_64-pc-windows-msvc
rustup home:  C:\Users\diamant3\.rustup

installed toolchains
--------------------

stable-x86_64-pc-windows-msvc
nightly-x86_64-pc-windows-msvc (default)

active toolchain
----------------

nightly-x86_64-pc-windows-msvc (default)
rustc 1.64.0-nightly (93ffde6f0 2022-07-23)
```

systeminfo:

```
PS C:\Users\diamant3\Desktop\psp-controller> systeminfo

Host Name:                 MSI
OS Name:                   Microsoft Windows 11 Home Single Language
OS Version:                10.0.22000 N/A Build 22000
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Owner:          diamant3
```


bug:

```
PS C:\Users\diamant3\Desktop\psp-controller> cargo psp
   Compiling psp v0.3.1
   Compiling cfg-if v0.1.10
   Compiling alloc v0.0.0 (C:\Users\diamant3\.rustup\toolchains\nightly-x86_64-pc-windows-msvc\lib\rustlib\src\rust\library\alloc)
error[E0433]: failed to resolve: could not find `unix` in `os`
 --> C:\Users\diamant3\.cargo\registry\src\github.com-1ecc6299db9ec823\psp-0.3.1\build.rs:1:20
  |
1 | use std::{env, os::unix::prelude::OsStrExt, path::Path};
  |                    ^^^^ could not find `unix` in `os`

   Compiling unwind v0.0.0 (C:\Users\diamant3\.rustup\toolchains\nightly-x86_64-pc-windows-msvc\lib\rustlib\src\rust\library\unwind)
error[E0599]: no method named `as_bytes` found for struct `OsString` in the current scope
  --> C:\Users\diamant3\.cargo\registry\src\github.com-1ecc6299db9ec823\psp-0.3.1\build.rs:15:10
   |
15 |         .as_bytes()
   |          ^^^^^^^^ method not found in `OsString`

Some errors have detailed explanations: E0433, E0599.
For more information about an error, try `rustc --explain E0433`.
error: could not compile `psp` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
PS C:\Users\diamant3\Desktop\psp-controller> 
```

--- a/psp/build.rs
+++ b/psp/build.rs
@@ -1,4 +1,4 @@
-use std::{env, os::unix::prelude::OsStrExt, path::Path};
+use std::{env, path::Path};
 
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
@@ -10,11 +10,10 @@ fn main() {
     }
 
     // Figure out whether to use the LTO libunwind, or the regular one.
-    let libunwind = if env::var_os("CARGO_ENCODED_RUSTFLAGS")
-        .expect("could not get `CARGO_ENCODED_RUSTFLAGS` variable")
-        .as_bytes()
-        .split(|b| *b == 0x1f)
-        .any(|flags| flags.starts_with(b"-Clinker-plugin-lto"))
+    let libunwind = if env::var("CARGO_ENCODED_RUSTFLAGS")
+        .unwrap()
+        .split('\x1f')
+        .any(|flags| flags.starts_with("-Clinker-plugin-lto"))
     {
         "./libunwind_lto.a"
     } else {

